.github/workflows/linux.yml

```diff
@@ -171,3 +177,9 @@ jobs:
         else
           exit 0
         fi
+
+    - name: Coveralls
+      uses: coverallsapp/github-action@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        path-to-lcov: ${{github.workspace}}/build/coverage_filtered.info
```

Comment:
I think you need to add `parallel: true`, since this is inside a matrix job (otherwise we get separate coverage results for deal.II 9.4 and master. Looks good otherwise, let's see if it already works for this PR.